docker/webui/Dockerfile-lb

```diff
@@ -1,9 +1,8 @@
+#!BuildTag: openqa_webui_lb
 FROM opensuse/leap:15.2
 LABEL maintainer Ivan Lausuch <ilausuch@suse.com>
 
-RUN zypper ar -f obs://devel:openQA/openSUSE_Leap_15.2 openQA && \
-    zypper ar -f obs://devel:openQA:Leap:15.2/openSUSE_Leap_15.2 openQA-perl-modules && \
-    zypper --gpg-auto-import-keys ref && \
+RUN zypper ar -f -G http://download.opensuse.org/repositories/devel:/openQA:/Leap:/15.2/openSUSE_Leap_15.2 openQA && \
```

Comment:
You can't simply replace two repos with one. devel:openQA provides os-autoinst and openQA whereas devel:openQA:Leap:15.2 provides the additional dependencies necessary for Leap 15.2. Doing it this way likely installs a very openQA version from Leap 15.2, not the devel:openQA version.

Reply:
I am not sure if this is applicable in this case, because only we are installing is nginx. But I am checking in https://github.com/os-autoinst/openQA/pull/3522 the mentioned solution

Reply:
uh, no. The next line says `zypper --non-interactive in --force-resolution openQA nginx`, so it's not just nginx